cv19ResSupportV3/V3/Gateways/HelpRequestGateway.cs

```diff
@@ -328,7 +330,9 @@ public HelpRequest PatchHelpRequest(int id, PatchHelpRequest command)
 
                 if (command.AssignedTo != null)
                 {
-                    rec.AssignedTo = command.AssignedTo;
+                    rec.CallHandlerId =
+                        _helpRequestsContext.CallHandlerEntities.FirstOrDefault(x => x.Name == command.AssignedTo)
+                        ?.Id;
```

Comment:
I see... I think we might need to ask Andrew for design on how that should look like, because when I think about **Bulk Assign** & the **Callbacks List** pages, they only contain the names of the Call Handlers, which would make it impossible to  tell the difference between the 2 CHs with identical name.

This is a lower probability case, of course, so I don't think we'd need to jump in immediately & code it, but having design on hand would be very useful for whenever we'd need to add that extra UI display in.

Reply:
Actually, I don't know why I'm typing this here, I'll just message you on Slack :joy: